### PR TITLE
build: set --build when configuring packages in depends

### LINKED ISSUE #16354: build: Review of depends tree configure flags
While doing depends builds, I noticed that there are many configure flags that weren't recognized by our packages. This leads to things like: https://github.com/bitcoin/bitcoin/pull/16183. This is most likely caused by either overzealous configure flag adding, or updating depends packages without looking at changes to accepted configure flags.

I propose that we:
1. Prune configure flags from depends that packages don't understand.
2. Add documentation on updating packages (namely that new configure flags should be checked).
3. (stretch-goal) Find out what `--without-` and `--disable-` configure flags we can supply to speed up depends builds.

CC @fanquake @theuni

### PR DESCRIPTION
**Pull request by fanquake**
After reading [#Specifying-Target-Triplets](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#Specifying-Target-Triplets) in the autoconf manuel, my understanding is that this change should mostly be a no-op, as `--build` defaults to the output of [`config.guess`](https://github.com/bitcoin/bitcoin/blob/master/depends/config.guess), however, this may be slightly more correct

>  For historical reasons, whenever you specify --host, be sure to specify --build too; this will be fixed in the future.

and will quell some warnings in depends (#16354), i.e:
> configure: WARNING: If you wanted to set the --build type, don't use --host.
    If a cross compiler is detected then cross compile mode will be used.

If anything, this also explicitly enables cross-compilation mode when `--host` differs from `--build`. As for "fixed in the future", this is the case for Autoconf 2.70+.

If we don't make this change, I think we should consider closing #16354. We've now got a good handle on our depends configure flags, and we can just leave the last few warnings as they are. It's also unlikely that we are going to add the `xorg-macros` package to depends.

Guix builds at 735610940c0cac72343ca247eb3062b2e09512a0:
```bash
bash-5.1# find output -type f -name *$(git rev-parse --short HEAD)*.* -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
afabf99650d57abd3df2c7e6253a3281f9814ada77d01c28ffb57c740b14fd8e  output/bitcoin-735610940c0c-aarch64-linux-gnu-debug.tar.gz
6e9dedced61489f7734c5387ee05823db68eb4b890fa05f11541c1b966576446  output/bitcoin-735610940c0c-aarch64-linux-gnu.tar.gz
5333a3c5978a61966251e56fedab8e059d9d8a1a7231a0908095bb373204a81f  output/bitcoin-735610940c0c-arm-linux-gnueabihf-debug.tar.gz
d28e1c166b96e58cf40b3f9801c3050c2785b25eeb261d44eeb31a32a75e134c  output/bitcoin-735610940c0c-arm-linux-gnueabihf.tar.gz
e00b0149c52e9207be1cfb56ec3b715b55f41691401095f7a7038ac124f08d80  output/bitcoin-735610940c0c-osx-unsigned.dmg
ae5052ad1a05916bc6656b034c169097471fd9bdba16b606e2bcb592c9395544  output/bitcoin-735610940c0c-osx-unsigned.tar.gz
8586a9b8b82a8832397fcd0756aa9649adc3e61ae579fb370b99fe406298199f  output/bitcoin-735610940c0c-osx64.tar.gz
d560a8781bae856b6593e9b731fa99de0a4ad7bd3e03084166dc98904ece9e7b  output/bitcoin-735610940c0c-powerpc64-linux-gnu-debug.tar.gz
a24cf28b7efd5ebdc892e7e8809a05b3405bb4222f7d2eaa32de265214c78f28  output/bitcoin-735610940c0c-powerpc64-linux-gnu.tar.gz
ad29421842dbbe96526eb597a7ff2970f0acae8c3e75dc6c2100716a2ba20606  output/bitcoin-735610940c0c-powerpc64le-linux-gnu-debug.tar.gz
ee7a8ccfd68cc2297bb96eaaacf7b3eb939a5e39cecc3c710dbe828518f1bfbd  output/bitcoin-735610940c0c-powerpc64le-linux-gnu.tar.gz
185bf4ea3b35072a8dd54aedbafc0892ec1e486c8ebd311f05f0fb47a19058cf  output/bitcoin-735610940c0c-riscv64-linux-gnu-debug.tar.gz
803078b15a9bbaa567176827639895e762edbe80844b80400f8a9581e5311d32  output/bitcoin-735610940c0c-riscv64-linux-gnu.tar.gz
d920327a32a42a5abadd247f0d3f3ee0ab0488a28892c11838d99bede6456723  output/bitcoin-735610940c0c-win-unsigned.tar.gz
9f1741a37e294af153d57ad4a6dad3e5f1cdf75f1588c86cb467abf0c177ae27  output/bitcoin-735610940c0c-win64-debug.zip
2d1a91552c0dd7bb400ce6c71d2e98de702743a97afe5f4b0945865f251f3aea  output/bitcoin-735610940c0c-win64-setup-unsigned.exe
8a70020bb0cb68516acf4b2cc89832d72373f31611bd075826a30a817a071cd2  output/bitcoin-735610940c0c-win64.zip
93b03837679474905538e96c44ba81ee5653795ad333dc00569b92c82dc6ea31  output/bitcoin-735610940c0c-x86_64-linux-gnu-debug.tar.gz
6ccd046a83ef5089667ad426875319beb83e2b2a5fb4d4459d5c5ce8d75f1a31  output/bitcoin-735610940c0c-x86_64-linux-gnu.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  output/src/bitcoin-735610940c0c.tar.gz
```

Gitian builds:
```bash
# macOS:
7d087749f610e5e3eca91c730d41afd4d404808e276242d05cedb89403de247f  bitcoin-735610940c0c-osx-unsigned.dmg
96d68b4f0042dc40f5e8e362fa84dcf8e122dcb566889feb48ad8a982847dd8b  bitcoin-735610940c0c-osx-unsigned.tar.gz
2dea72cbe5c3e228babd88b516a6f43b0e040e9510ea95b62f07fc0815bf8d22  bitcoin-735610940c0c-osx64.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
478dbda9d551a8bed985c61c625df3f543afab26989a52bc44d5a4745715af81  bitcoin-core-osx-22-res.yml

# Windows:
c31a5118dfa21259b9868e2c4858c635becb786c2ee8af92302da7811a1efe9b  bitcoin-735610940c0c-win-unsigned.tar.gz
5c171ec3aee830abc8ae843d674d5fe7a57420e782ee765b6992914ae5b7671a  bitcoin-735610940c0c-win64-debug.zip
ca010ced982f75d111e01f3759b961deb83e7bf511b1ee9c9041f4830570f2a0  bitcoin-735610940c0c-win64-setup-unsigned.exe
62514b01281f81af6057fb12cc5e3813c1d8f6532d0d8ac8f6f2909a595aa1dd  bitcoin-735610940c0c-win64.zip
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
e26b55d8302a419f594396db6ff441aaeb5e275e02fde7b1b4eb092270f902e7  bitcoin-core-win-22-res.yml

# Linux:
e5be4deb91cb372de484468c0575a04f907f982e35a9841659bb213540f770d0  bitcoin-735610940c0c-aarch64-linux-gnu-debug.tar.gz
6169e822679ece2bb6af7027362a6f373eb939eef9d89edc58c21190b4083c7d  bitcoin-735610940c0c-aarch64-linux-gnu.tar.gz
aee5308767c5c930853a80ab94325b2a4ab3dbe9a7a20c7a9d60d57af2b5383f  bitcoin-735610940c0c-arm-linux-gnueabihf-debug.tar.gz
a9cf6e56d859fffd5a2b5f1e6360515bc6b06b048776b93cc08954859457a251  bitcoin-735610940c0c-arm-linux-gnueabihf.tar.gz
7617d85c358e6614fd40026a8d346f9edba5b5139e3daade2ec2aec72aed3a1f  bitcoin-735610940c0c-powerpc64-linux-gnu-debug.tar.gz
f76e46a8b130812541ebcaaba173b4cbe2ec1781e348579b44f96e9244cc2475  bitcoin-735610940c0c-powerpc64-linux-gnu.tar.gz
19ada7abb801422bd7772d7ee559004cab11a46141a747aabd27085e8c3d200d  bitcoin-735610940c0c-powerpc64le-linux-gnu-debug.tar.gz
f3a6bbefe9f47b3960e3b9eeb3f740ecddf8773ae76961ced6a3cdf82503e37f  bitcoin-735610940c0c-powerpc64le-linux-gnu.tar.gz
42ca333e8f82b4ec0b2354efc7c4f1c27ed32d60c35b88e33d26833419f19e69  bitcoin-735610940c0c-riscv64-linux-gnu-debug.tar.gz
dde930d9134fb5be1374014db3f6f503b614ae801a1d77f6f4b82a5f11eb2518  bitcoin-735610940c0c-riscv64-linux-gnu.tar.gz
82e1b4cc8aac309a99458cc27fd0285215a8af53780094c4990303ce8948cf0c  bitcoin-735610940c0c-x86_64-linux-gnu-debug.tar.gz
ee8caad2ccddd8eaf8aa2d19040d859130168183355b09a2a0f77a59a97fcaee  bitcoin-735610940c0c-x86_64-linux-gnu.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
d90545b13226a338c994d9e538f468a6fafd694273bf71e14614feeaf5727ad4  bitcoin-core-linux-22-res.yml
```

### CODE REVIEW

**Review by jarolrod — COMMENTED** [review]
contributing GUIX build hashes

```
afde0433a26088bf4a5c6966bc88913fd17ff6ceb2a6a629640c24f8823a7b0e  output/bitcoin-4f91544c4d29-aarch64-linux-gnu-debug.tar.gz
cd5e4caa8a890a15465511b34d5fbceb2c742e43fd3650eb010f7653a7ae7ecb  output/bitcoin-4f91544c4d29-aarch64-linux-gnu.tar.gz
b9afd9d79e7f28b1b29b01039d00c3643b0d32c169415b327d07a70bcd1e2aeb  output/bitcoin-4f91544c4d29-arm-linux-gnueabihf-debug.tar.gz
c7fdeb894dd9360f068e63f36ecb7617b2f1ed6e8805cb4f84f7bc166e015017  output/bitcoin-4f91544c4d29-arm-linux-gnueabihf.tar.gz
11ab0c284693a6c366f1c23033a1dabe5d6a75e726248b7c08492b268d6698b5  output/bitcoin-4f91544c4d29-osx-unsigned.dmg
bd3c52b921f27b9bb74cbf0d75ed65dc4f1381e207a1df3cb0f742d587fb6bb7  output/bitcoin-4f91544c4d29-osx-unsigned.tar.gz
e8b8639880bba05982a8c10005e8f7393e8fdfdd5dcb613a050607b3e9e09db0  output/bitcoin-4f91544c4d29-osx64.tar.gz
8e3129f7e36e2e58332d17aebf9ede60dd077542da58ee67298f28e39382ebb3  output/bitcoin-4f91544c4d29-powerpc64-linux-gnu-debug.tar.gz
3faa2fb13b8566bf9ad7295fa137ce0b11d17c60617e07f88f5936c57d4f0e59  output/bitcoin-4f91544c4d29-powerpc64-linux-gnu.tar.gz
a61bc8be61323f313766eb9284a78bdcc5cbe850a3db33665a29fbb051d5c6c2  output/bitcoin-4f91544c4d29-powerpc64le-linux-gnu-debug.tar.gz
ec5a8a982d62bb7a8b4a344e761712496c70d1d1ecb08f5e9718a512f97d9a8b  output/bitcoin-4f91544c4d29-powerpc64le-linux-gnu.tar.gz
33fb9a7e0a816f5d9e2e9e846c08ccb5ac485ab69a201ecd10f59b015158ced4  output/bitcoin-4f91544c4d29-riscv64-linux-gnu-debug.tar.gz
248267833015b8ee99209be655a84b090041169569109be7ae1a97cefda86806  output/bitcoin-4f91544c4d29-riscv64-linux-gnu.tar.gz
52aaf8313e4babbe2f774852d65635a1cbbe0b28730d9204bb1efce068f31025  output/bitcoin-4f91544c4d29-win-unsigned.tar.gz
c207d38a863bfd2544375fb9aa943dceafc13ab1f1ce4318aa4ab8245694c665  output/bitcoin-4f91544c4d29-win64-debug.zip
07acd38150f7add44ce00e3d5add134d4bad3f097c12f73135775e3a8f89c3eb  output/bitcoin-4f91544c4d29-win64-setup-unsigned.exe
83be8c289be6bea72b5b99e1edbc754ec8d8af1809a6ab5471e4f2255a16416c  output/bitcoin-4f91544c4d29-win64.zip
fa4a8814116dc997b6ed217617b7f0ce26161a8dd6cafc3c1d82a9a68392be8d  output/bitcoin-4f91544c4d29-x86_64-linux-gnu-debug.tar.gz
1072a11e163798e8dc572786876d41cdc5d69f36084aa04791d83f60b84bc127  output/bitcoin-4f91544c4d29-x86_64-linux-gnu.tar.gz
b85ffb7d4f6e7fe4ee98f51a76e545089c49db1acfe031bbac6dacd659182224  output/src/bitcoin-4f91544c4d29.tar.gz
```

**Comment by hebasto** [comment]
Concept ACK. The same idea was in my TODO list.

File: depends/funcs.mk [hunk 1]
```diff
@@ -140,7 +140,7 @@ $(1)_config_env+=CMAKE_MODULE_PATH=$($($(1)_type)_prefix)/lib/cmake
 $(1)_config_env+=PATH=$(build_prefix)/bin:$(PATH)
 $(1)_build_env+=PATH=$(build_prefix)/bin:$(PATH)
 $(1)_stage_env+=PATH=$(build_prefix)/bin:$(PATH)
-$(1)_autoconf=./configure --host=$($($(1)_type)_host) --prefix=$($($(1)_type)_prefix) $$($(1)_config_opts) CC="$$($(1)_cc)" CXX="$$($(1)_cxx)"
+$(1)_autoconf=./configure --build=$(build) --host=$($($(1)_type)_host) --prefix=$($($(1)_type)_prefix) $$($(1)_config_opts) CC="$$($(1)_cc)" CXX="$$($(1)_cxx)"
```
Comment (hebasto) [inline depends/funcs.mk:143]: From Autoconf [manual](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#Specifying-Target-Triplets):
>  It defaults to the result of running `config.guess`.

In our code https://github.com/bitcoin/bitcoin/blob/e0bc27a14ceb18bfe3dd9ebdae6bef447c4ec922/depends/Makefile#L69

Could this cause any subtle problems?

**Review by hebasto — COMMENTED** [review]
Tested 4f91544c4d2910ae0fd3a356e3c2e785ab1c818c on Linux Mint 20.1 (x86_64).

On master:
```
$ make -C depends libXau_configured
...
Configuring libXau...
configure: WARNING: if you wanted to set the --build type, don't use --host.
    If a cross compiler is detected then cross compile mode will be used
...


$ make -C depends bdb_configured
...
Configuring bdb...
configure: WARNING: If you wanted to set the --build type, don't use --host.
    If a cross compiler is detected then cross compile mode will be used.
...
```

No such warnings with this PR.

**Comment by dongcarl** [comment]
Concept ACK!

-----

Wondering if it would be better to place the context documentation in the file rather than in the git message. Locating git messages/PRs can become difficult if a line gets changed often and moved around.

**Comment by fanquake** [comment]
> Wondering if it would be better to place the context documentation in the file rather than in the git message.

I've updated this to add some documentation inline. Let me know if it is explanatory enough.

**Comment by dongcarl** [comment]
Code Review ACK 735610940c0cac72343ca247eb3062b2e09512a0

**Review by hebasto — APPROVED** [review]
ACK 735610940c0cac72343ca247eb3062b2e09512a0, tested on Linux Mint 20.1 (x86_64).

**Comment by hebasto** [comment]
#### Guix builds:
```
$ find output -type f -name *$(git rev-parse --short HEAD)*.* -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
afabf99650d57abd3df2c7e6253a3281f9814ada77d01c28ffb57c740b14fd8e  output/bitcoin-735610940c0c-aarch64-linux-gnu-debug.tar.gz
6e9dedced61489f7734c5387ee05823db68eb4b890fa05f11541c1b966576446  output/bitcoin-735610940c0c-aarch64-linux-gnu.tar.gz
5333a3c5978a61966251e56fedab8e059d9d8a1a7231a0908095bb373204a81f  output/bitcoin-735610940c0c-arm-linux-gnueabihf-debug.tar.gz
d28e1c166b96e58cf40b3f9801c3050c2785b25eeb261d44eeb31a32a75e134c  output/bitcoin-735610940c0c-arm-linux-gnueabihf.tar.gz
e00b0149c52e9207be1cfb56ec3b715b55f41691401095f7a7038ac124f08d80  output/bitcoin-735610940c0c-osx-unsigned.dmg
ae5052ad1a05916bc6656b034c169097471fd9bdba16b606e2bcb592c9395544  output/bitcoin-735610940c0c-osx-unsigned.tar.gz
8586a9b8b82a8832397fcd0756aa9649adc3e61ae579fb370b99fe406298199f  output/bitcoin-735610940c0c-osx64.tar.gz
d560a8781bae856b6593e9b731fa99de0a4ad7bd3e03084166dc98904ece9e7b  output/bitcoin-735610940c0c-powerpc64-linux-gnu-debug.tar.gz
a24cf28b7efd5ebdc892e7e8809a05b3405bb4222f7d2eaa32de265214c78f28  output/bitcoin-735610940c0c-powerpc64-linux-gnu.tar.gz
ad29421842dbbe96526eb597a7ff2970f0acae8c3e75dc6c2100716a2ba20606  output/bitcoin-735610940c0c-powerpc64le-linux-gnu-debug.tar.gz
ee7a8ccfd68cc2297bb96eaaacf7b3eb939a5e39cecc3c710dbe828518f1bfbd  output/bitcoin-735610940c0c-powerpc64le-linux-gnu.tar.gz
185bf4ea3b35072a8dd54aedbafc0892ec1e486c8ebd311f05f0fb47a19058cf  output/bitcoin-735610940c0c-riscv64-linux-gnu-debug.tar.gz
803078b15a9bbaa567176827639895e762edbe80844b80400f8a9581e5311d32  output/bitcoin-735610940c0c-riscv64-linux-gnu.tar.gz
d920327a32a42a5abadd247f0d3f3ee0ab0488a28892c11838d99bede6456723  output/bitcoin-735610940c0c-win-unsigned.tar.gz
9f1741a37e294af153d57ad4a6dad3e5f1cdf75f1588c86cb467abf0c177ae27  output/bitcoin-735610940c0c-win64-debug.zip
2d1a91552c0dd7bb400ce6c71d2e98de702743a97afe5f4b0945865f251f3aea  output/bitcoin-735610940c0c-win64-setup-unsigned.exe
8a70020bb0cb68516acf4b2cc89832d72373f31611bd075826a30a817a071cd2  output/bitcoin-735610940c0c-win64.zip
93b03837679474905538e96c44ba81ee5653795ad333dc00569b92c82dc6ea31  output/bitcoin-735610940c0c-x86_64-linux-gnu-debug.tar.gz
6ccd046a83ef5089667ad426875319beb83e2b2a5fb4d4459d5c5ce8d75f1a31  output/bitcoin-735610940c0c-x86_64-linux-gnu.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  output/src/bitcoin-735610940c0c.tar.gz
```

**Comment by hebasto** [comment]
#### Gitian builds:
- Linux
```Generating report
e5be4deb91cb372de484468c0575a04f907f982e35a9841659bb213540f770d0  bitcoin-735610940c0c-aarch64-linux-gnu-debug.tar.gz
6169e822679ece2bb6af7027362a6f373eb939eef9d89edc58c21190b4083c7d  bitcoin-735610940c0c-aarch64-linux-gnu.tar.gz
aee5308767c5c930853a80ab94325b2a4ab3dbe9a7a20c7a9d60d57af2b5383f  bitcoin-735610940c0c-arm-linux-gnueabihf-debug.tar.gz
a9cf6e56d859fffd5a2b5f1e6360515bc6b06b048776b93cc08954859457a251  bitcoin-735610940c0c-arm-linux-gnueabihf.tar.gz
7617d85c358e6614fd40026a8d346f9edba5b5139e3daade2ec2aec72aed3a1f  bitcoin-735610940c0c-powerpc64-linux-gnu-debug.tar.gz
f76e46a8b130812541ebcaaba173b4cbe2ec1781e348579b44f96e9244cc2475  bitcoin-735610940c0c-powerpc64-linux-gnu.tar.gz
ccaebafe26ac5effcd40efc76289fa3c7ed6a5d1f18bdfc20cf27ab7bcfc8291  bitcoin-735610940c0c-powerpc64le-linux-gnu-debug.tar.gz
4c1844280ec790b2fc64808f4bf10df43c8cbdbb3f8be12a1382c3205bbe7e07  bitcoin-735610940c0c-powerpc64le-linux-gnu.tar.gz
42ca333e8f82b4ec0b2354efc7c4f1c27ed32d60c35b88e33d26833419f19e69  bitcoin-735610940c0c-riscv64-linux-gnu-debug.tar.gz
dde930d9134fb5be1374014db3f6f503b614ae801a1d77f6f4b82a5f11eb2518  bitcoin-735610940c0c-riscv64-linux-gnu.tar.gz
82e1b4cc8aac309a99458cc27fd0285215a8af53780094c4990303ce8948cf0c  bitcoin-735610940c0c-x86_64-linux-gnu-debug.tar.gz
ee8caad2ccddd8eaf8aa2d19040d859130168183355b09a2a0f77a59a97fcaee  bitcoin-735610940c0c-x86_64-linux-gnu.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
e6c0a9e4fca206a96aedd4d16579e6e9344e8ff645bd747cc851779ce702eb98  bitcoin-core-linux-22-res.yml
Done.
```
- Windows
```
Generating report
c31a5118dfa21259b9868e2c4858c635becb786c2ee8af92302da7811a1efe9b  bitcoin-735610940c0c-win-unsigned.tar.gz
5c171ec3aee830abc8ae843d674d5fe7a57420e782ee765b6992914ae5b7671a  bitcoin-735610940c0c-win64-debug.zip
ca010ced982f75d111e01f3759b961deb83e7bf511b1ee9c9041f4830570f2a0  bitcoin-735610940c0c-win64-setup-unsigned.exe
62514b01281f81af6057fb12cc5e3813c1d8f6532d0d8ac8f6f2909a595aa1dd  bitcoin-735610940c0c-win64.zip
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
ac985f392dda28d5ca3f9db154ebae236e7b2336b68726e6208c5dd5a8b8c9cf  bitcoin-core-win-22-res.yml
Done.
```
- macOS
```
Generating report
f6391c0774f751a22cef8c3bbada9ab290c1140fb844206d1446727d2389cfd6  bitcoin-735610940c0c-osx-unsigned.dmg
36a9dc7435d6128ae06ae2c07c6e3366f3947f222f5ee87fbd7b92d9cd9237cb  bitcoin-735610940c0c-osx-unsigned.tar.gz
aba38636e4f44a7f36be2f1cd76443ff3a703199315430d294334e915b627402  bitcoin-735610940c0c-osx64.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
3ba5f361ed8e95e52629e0cbf18d0739869cd56c00c252312dfbffc30bf1261a  bitcoin-core-osx-22-res.yml
Done.
```

**Comment by hebasto** [comment]
Another Gitian macOS build:
```
Generating report
7d087749f610e5e3eca91c730d41afd4d404808e276242d05cedb89403de247f  bitcoin-735610940c0c-osx-unsigned.dmg
96d68b4f0042dc40f5e8e362fa84dcf8e122dcb566889feb48ad8a982847dd8b  bitcoin-735610940c0c-osx-unsigned.tar.gz
2dea72cbe5c3e228babd88b516a6f43b0e040e9510ea95b62f07fc0815bf8d22  bitcoin-735610940c0c-osx64.tar.gz
12771acb17de84cda8e81c59704c00b1d4725ff51fa84c9067dc61a8dc795bdc  src/bitcoin-735610940c0c.tar.gz
d4622f430985538d35e3a83eed9a8a48739cd2251a67b4cb916ba45493eb1149  bitcoin-core-osx-22-res.yml
Done.
```

**Pull request merged**